src/bermudan.cpp

```diff
@@ -162,8 +165,8 @@ Rcpp::List bermudanSwaptionEngine(Rcpp::List rparam,
     QuantLib::Rate dummyFixedRate = 0.03;
     boost::shared_ptr<QuantLib::IborIndex> indexSixMonths(new QuantLib::Euribor6M(rhTermStructure));
         
-    QuantLib::Date startDate = calendar.advance(settlementDate, 1, QuantLib::Years, floatingLegConvention);
-    QuantLib::Date maturity = calendar.advance(startDate, 5, QuantLib::Years, floatingLegConvention);
+    //QuantLib::Date startDate = calendar.advance(settlementDate, 1, QuantLib::Years, floatingLegConvention);  //took out hard coded 
+    //QuantLib::Date maturity = calendar.advance(startDate, 5, QuantLib::Years, floatingLegConvention);         //dates
```

Comment:
I didn¹t bother to make it backward compatible

From:  Dirk Eddelbuettel <notifications@github.com>
Reply-To:  eddelbuettel/rquantlib
<reply+007042a2c30dfa310604e6556eca0153d77ba0c90f8543e992cf00000001130d2c9b9
2a170ce036ccc93@reply.github.com>
Date:  Friday, March 25, 2016 at 12:43 PM
To:  eddelbuettel/rquantlib <rquantlib@noreply.github.com>
Cc:  terry leitch <tleitch@uchicago.edu>
Subject:  Re: [rquantlib] Modified bermudan option module to allow for
swaptions other than 1 y (#34)


In src/bermudan.cpp
<https://github.com/eddelbuettel/rquantlib/pull/34#discussion_r57461907> :
> @@ -162,8 +165,8 @@ Rcpp::List bermudanSwaptionEngine(Rcpp::List rparam,
>      QuantLib::Rate dummyFixedRate = 0.03;
>      boost::shared_ptr<QuantLib::IborIndex> indexSixMonths(new
QuantLib::Euribor6M(rhTermStructure));
>          
> -    QuantLib::Date startDate = calendar.advance(settlementDate, 1,
QuantLib::Years, floatingLegConvention);
> -    QuantLib::Date maturity = calendar.advance(startDate, 5, QuantLib::Years,
floatingLegConvention);
> +    //QuantLib::Date startDate = calendar.advance(settlementDate, 1,
QuantLib::Years, floatingLegConvention);  //took out hard coded
> +    //QuantLib::Date maturity = calendar.advance(startDate, 5,
QuantLib::Years, floatingLegConvention);         //dates
Not sure how those slipped in / survived for a decade.  Ooops.


You are receiving this because you modified the open/close state.
Reply to this email directly or view it on GitHub
<https://github.com/eddelbuettel/rquantlib/pull/34/files/f64268ae91b4bae7767
838a671cbf2f65186a171#r57461907>